src/ontology/mondo-edit.obo

```diff
@@ -619540,6 +619542,14 @@ relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-7638-4659
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/9174" xsd:anyURI
 
+[Term]
+id: MONDO:1060150
+name: NOTCH1-related AOS spectrum disorder
+def: "A spectrum of cardiac and extracardiac phenotypes caused by a disease-causing variant in the NOTCH1 gene, inherited in an autosomal dominant manner. Affected individuals may present with congenital heart defects, bicuspid aortic valve, aortic valve stenosis, thoracic aortic aneurysm or dissection, anomalies in brain structure, intracranial or posterior circulation vascular anomalies, cutaneous vascular malformations, cutis marmorata, and/or a phenotype compatible with Adams-Oliver syndrome (i.e. cutis aplasia, terminal limb defects, skull ossification defects). Individuals with pathogenic variants in NOTCH1 may also be asymptomatic due to incomplete penetrance and extensive intrafamilial variability." [https://clinicalgenome.org/affiliation/40104/, https://orcid.org/0009-0007-6180-2639, PMID:25963545]
```

Comment:
I know we need to put together an official pattern for "spectrum", but the definition as such currently means "a spectrum of phenotypes". I think the definition should starts with "a disease characterized by a spectrum for phenotypes

I would put the following in a comment: "Individuals with pathogenic variants in NOTCH1 may also be asymptomatic due to incomplete penetrance and extensive intrafamilial variability."
This doesn't really have a relevance in the definition, and it is a bit confusing (eg if a patient is asymptomatic, do they have this disease?)